File: 1051-height-checker/1051-height-checker.java
class Solution {
    public int heightChecker(int[] heights) {
        int mismatch=0;
        int []sortedHeights=Arrays.copyOf(heights,heights.length);
        Arrays.sort(heights);
        for(int i=0;i<heights.length;i++){
            if(heights[i]!=sortedHeights[i]){
                mismatch++;
            }
        }
        return mismatch;
    }
}

File: 1051-height-checker/1051-height-checker.cpp
class Solution {
public:
    int heightChecker(vector<int>& heights) {
        int count=0;
        vector<int> sortedHeights=heights;
        sort(sortedHeights.begin(),sortedHeights.end());
        for(int i=0;i<heights.size();i++){
            if(heights[i]!=sortedHeights[i]){
                count++;
            }
        }
        return count;
        
    }
};